eulerpi/examples/heat/heat.py

```diff
@@ -104,17 +105,31 @@ def forward(cls, param: np.ndarray) -> np.ndarray:
 
         solution = cls.perform_simulation(kappa=param)
 
-        # the indices of the wanted evaluation points
-        evaluation_indices = jnp.multiply(
-            cls.evaluation_points,
-            jnp.array(solution.shape),
-        ).astype(int)
-
-        solution_at_evaluation_points = jnp.array(
-            solution[
-                evaluation_indices[:, 0],
-                evaluation_indices[:, 1],
-            ]
+        # linearly interpolate the solution at the evaluation points, use own interpolation function as jax doesn't support scipy.interpolate.interp2d and doesn't provide a 2d interpolation function
+        x = jnp.linspace(0, cls.plate_length[0], cls.num_grid_points)
+        y = jnp.linspace(0, cls.plate_length[1], cls.num_grid_points)
+        X, Y = jnp.meshgrid(x, y)
```

Comment:
X, Y are unused -> Remove them?

Reply:
Yep, just saw that a second too late, will do!